e2e/tests/uniswap_trade_test.rs

```diff
@@ -211,18 +207,20 @@ async fn test_with_ganache() {
 
     // Drive orderbook in order to check the removal of settled order_b
     orderbook.run_maintenance(&gp_settlement).await.unwrap();
-    let placement = client
-        .get(&format!(
-            "{}{}{}",
-            API_HOST,
-            ORDER_PLACEMENT_ENDPOINT,
-            order_b.uid(&trader_b.address())
-        ))
-        .send()
-        .await;
-    assert_eq!(placement.unwrap().status(), 404);
+    let orders = create_orderbook_api().get_orders().await.unwrap();
+    assert!(orders
```

Comment:
Would it be safe to just check that it's empty, given that both orders should be matched?

Reply:
I guess that should be fine too because we clear the order database in the test. Will change to that.